CHANGELOG.md

```diff
@@ -8,6 +8,7 @@ Enhancement:
 
 - Enhanced to support Photon OS 4.0 and 5.0 based appliances. [GH-202](https://github.com/vmware/powershell-module-for-vmware-cloud-foundation-password-management/pull/202)
 - Enhanced `Get-PasswordPolicyDefault` to support VMware Cloud Foundation 5.2. [GH-204](https://github.com/vmware/powershell-module-for-vmware-cloud-foundation-password-management/pull/204), [GH-206](https://github.com/vmware/powershell-module-for-vmware-cloud-foundation-password-management/pull/206)
+- Added Automation to test Aria components.  [GH-204](https://github.com/vmware/powershell-module-for-vmware-cloud-foundation-password-management/pull/204), [GH-210](https://github.com/vmware/powershell-module-for-vmware-cloud-foundation-password-management/pull/210)
```

Comment:
This comment is added for an version that has already shipped. It's okay to remove this addition.

Reply:
Resolved in 9727035.